### PR TITLE
[compute] fixes to handle no available compute and other options

### PR DESCRIPTION
**Pull request by amitsrivastava**
Change-Id: I2b9b408e0f342a2134da7c2f65ee6b9cdcc4d340

### CODE REVIEW

File: apps/beeswax/src/beeswax/server/dbms.py [hunk 1]
```diff
@@ -338,8 +338,8 @@ def get_query_server_config_via_connector(connector):
       # For connectors/computes, the auth details are not available
       # from configs and needs patching before submitting requests
       'principal': 'TODO',
-      'auth_username': None,
-      'auth_password': '',
+      'auth_username': compute['options'].get('auth_username'),
+      'auth_password': compute['options'].get('auth_password', 'hue'),
```
Comment (Harshg999) [inline apps/beeswax/src/beeswax/server/dbms.py:342]: From where is this default `hue` value coming from?

Comment (amitsrivastava) [reply apps/beeswax/src/beeswax/server/dbms.py:342]: this is our current default from our configs. Won't really get used in most cases are we are impersonating and sending the `auth_username` only.

**Review by Harshg999 — APPROVED** [review]

**Review by quadoss — APPROVED** [review]
LGTM

**Review by agl29 — APPROVED** [review]
LGTM!

**Pull request merged**